src/components/subscriptions/licenses/LicenseManagementTable/index.jsx

```diff
@@ -0,0 +1,296 @@
+      recent: userRecentAction(user),
+    })),
+    [users],
+  );
+
+  // Row action button functions
+  const rowRemindOnClick = (remindUser) => {
+    setRemindModal({
+      open: true,
+      users: [remindUser],
+    });
+  };
+  const rowRevokeOnClick = (revokeUser) => {
+    setRevokeModal({
+      open: true,
+      users: [revokeUser],
+    });
+  };
+
+  // Successful action modal callback
+  const onRemindSuccess = () => {
+    forceRefreshUsers();
+    setRemindModal(modalZeroState);
+    addToast(`User${revokeModal.users.length > 1 ? 's' : ''} successfully reminded`);
+  };
+  const onRevokeSuccess = () => {
+    // refresh subscription and user data to get updated revoke count and revoked user list
+    forceRefreshSubscription();
+    forceRefreshUsers();
+    forceRefreshOverview();
+    setRevokeModal(modalZeroState);
+    addToast(`License${revokeModal.users.length > 1 ? 's' : ''} successfully revoked`);
+  };
+
+  // Bulk Action buttons
+  const bulkRemindOnClick = (usersToRemind) => {
+    setRemindModal({
+      open: true,
+      users: usersToRemind,
+    });
+  };
+  const bulkRevokeOnClick = (usersToRevoke) => {
+    setRevokeModal({
+      open: true,
+      users: usersToRevoke,
+    });
+  };
+
+  return (
+    <>
+      {(users.count < 1 && userStatusFilter === null) && <SubscriptionZeroStateMessage /> }
+
+      <DataTable
+        showFiltersInSidebar={showFiltersInSidebar}
+        isFilterable
+        manualFilters
+        isSelectable
+        manualSelectColumn={selectColumn}
+        SelectionStatusComponent={DataTable.ControlledSelectionStatus}
+        defaultColumnValues={{ Filter: TextFilter }}
+        isPaginated
+        manualPagination
+        itemCount={users.count}
+        pageCount={users.numPages || 1}
+        // eslint-disable-next-line no-unused-vars
+        tableActions={(i) => <DownloadCsvButton />}
+        initialState={{
+          pageSize: PAGE_SIZE,
+          pageIndex: DEFAULT_PAGE - 1,
+        }}
+        initialTableOptions={{
+          getRowId: row => row.id,
+        }}
+        fetchData={fetchData}
+        data={data}
+        columns={[
+          {
+            Header: 'Email address',
+            accessor: 'emailLabel',
+          },
+          {
+            Header: 'Status',
+            accessor: 'statusBadge',
+            Filter: CheckboxFilter,
+            filter: 'includesValue',
+            filterChoices: [{
+              name: 'Active',
+              number: overview.activated,
+              value: ACTIVATED,
+            },
+            {
+              name: 'Pending',
+              number: overview.assigned,
+              value: ASSIGNED,
+            },
+            {
+              name: 'Revoked',
+              number: overview.revoked,
+              value: REVOKED,
+            }],
+          },
+          {
+            Header: 'Recent action',
+            accessor: 'recent',
```

Comment:
nit: maybe `recentAction` so it's a bit more clear what `recent` is referring when accessed elsewhere?